addons/dmx-adapter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "id": "dmx-adapter",
+  "name": "Dmx",
+  "description": "Connect your DMX devices",
+  "author": "Tim Hellhake",
+  "homepage_url": "https://github.com/tim-hellhake/dmx-adapter#readme",
+  "license_url": "https://raw.githubusercontent.com/tim-hellhake/dmx-adapter/master/LICENSE",
+  "primary_type": "adapter",
+  "packages": [
+    {
+      "architecture": "linux-arm",
+      "language": {
+        "name": "nodejs",
```

Comment:
This should be `binary`, not `nodejs`.

Reply:
I wasn't if the addon proxy knows `binary`.

Reply:
I wasn’t either but it looks like it works. 😆